### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vahidalvandi**
add ladumor/laravel-pwa in dev composer

### CODE REVIEW

**Comment by shailesh-ladumor** [comment]
@vahidalvandi  why it should be in dev?

**Comment by vahidalvandi** [comment]
Because it does not need to be installed in the production environment
composer u --no-dev

**Comment by shailesh-ladumor** [comment]
ok, I will test and merge today!

File: README.md [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 Install the package by the following command,
```
Comment (shailesh-ladumor) [inline README.md:19]:
```suggestion
Install the package by the following command, (try without `--dev` if you want to install it on production environment)
```

**Review by shailesh-ladumor — APPROVED** [review]

**Review by shailesh-ladumor — APPROVED** [review]

**Pull request merged**